plugfest/index.html

```diff
@@ -1,239 +1,274 @@
--->
-<ul>
-    <li><a href="./playground/">Playground</a></li>
-</ul>
-
-
-<h2>WebUI <!-- and TD Transformer--></h2>
-<ul>
-	<li>Browsified node-wot, see <a href="http://plugfest.thingweb.io/webui/">http://plugfest.thingweb.io/webui/</a></li>
-	<li>
-		Browser Examples
-		<ul>
-			<li><a href="http://plugfest.thingweb.io/examples/smart-coffee-machine.html">Smart Coffee Machine</a></li>
-			<li><a href="http://plugfest.thingweb.io/examples/counter.html">Counter</a></li>
-		</ul>
-	</li>
-	<li>
-		JavaFX-based UI (see <a href="https://github.com/danielpeintner/wot-fxui">WoT-FXUI</a>)
-		<ul>
-			<li>See <a href="http://plugfest.thingweb.io:8088/test/fullscreen/default">running Web-UI instance</a> in browser without any plugin</li>
-			<li>Possible as a local desktop application and an Android/iOS app also</li>
-		</ul>
-	</li>
-
-
-	<!--
-    <li><a href="http://plugfest.thingweb.io/webui/">Thing Client UI</a> (Note: needs to be updated for Prague PlugFest)</li>
-	-->
-	<!-- <li><a href="/td-transformer/">Thing Description Transformer</a></li> -->
-</ul>
-		
-      </div>
-      
-      <div class="footer-shape">
-        <img src="./img/footer-shape.jpg" width="100%"/>
-      </div>
-      <footer>
-        <div class="links">
-          <ul>
-            <li>
-              <a href="http://www.eclipse.org">Eclipse Foundation</a>
-            </li>
-            <li>|</li>
-            <li>
-              <a href="http://www.eclipse.org/legal/privacy.php">Privacy policy</a>
-            </li>
-            <li>|</li>
-            <li>
-              <a href="http://www.eclipse.org/legal/termsofuse.php">Terms of use</a>
-            </li>
-            <li>|</li>
-            <li>
-              <a href="http://www.eclipse.org/legal/copyright.php">Copyright</a>
-            </li>
-            <li>|</li>
-            <li>
-              <a href="http://www.eclipse.org/legal">Legal</a>
-            </li>
-          </ul>
-        </div>
-        <div>
-          <p>Copyrights ©  2023 Eclipse Thingweb. All Rights Reserved.</p>
-        </div>
-      </footer>
-    </body>
-</html>
+    <ul>
+      <li>
+        <b>Counter:</b> It has a count property that can be read or observed and can be incremented or decremented via separate actions. 
+        It is also possible to reset the count the value, obtain when the last change occurred, subscribe to a change in the count value or get the count value as an image.
+        TDs:
+        <ul>
+          <li>HTTP: http://plugfest.thingweb.io:8083/counter</li>
+          <li>CoAP: coap://plugfest.thingweb.io:5683/counter</li>
+        </ul>
+      </li>
+      <li>
+        <b>TestThing:</b> This Thing exists primarily for testing different data schemas and payload formats. 
+        It also has events attached to affordances that notify when a value changes.
+        TDs
+        <ul>
+          <li>HTTP: http://plugfest.thingweb.io:8083/testthing</li>
```

Comment:
for the HTTP case I would add an actual link so that people can click on it in a browser